fit/core-reference/src/main/resources/log.properties

```diff
@@ -0,0 +1,17 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why this file? Haven't you already added `${log.directory}` to `fit/core-reference/src/test/resources/core-test.properties`?